docs: update styles (#1883)
apollographql/apollo-cli: docs/source/commands/dev.mdx

While you're making local changes to an individual subgraph, you can use the `rover dev` command to start a local router instance and test out the effects of those changes on your entire supergraph.

> <Note>
> 
> The `rover dev` command is for local development only. Do not run `rover dev` in production.
> 
> </Note>

Reviewer: Feels more like a caution than a note? 

```suggestion
<Caution>

Don't run `rover dev` in production. It's for local development only.

</Caution>
```
